addons/calendar/static/tests/activity_tests.js

```diff
@@ -35,3 +35,42 @@ QUnit.test("activity click on Reschedule", async () => {
     await click(".btn", { text: "Reschedule" });
     await contains(".o_calendar_view");
 });
+
+QUnit.test("activity with event click on cancel", async (assert) => {
```

Comment:
I think it would be better to give a name that explains the real purpose of the test. Here, the important part is actually what happens once you clicked on cancel: the activity is removed so it might be better  to name it like `Can cancel activity linked to an event` or something like that.